gcp.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+git commit -am "$1"
+git push
```

Comment:
No mistake.  Useful script.  What makes it private?

On Saturday, April 11, 2015, CharliePoole <notifications@github.com> wrote:

> In gcp.sh <https://github.com/nunit/nunit/pull/586#discussion_r28156075>:
>
> > @@ -0,0 +1,3 @@
> > +#!/usr/bin/env bash
> > +git commit -am "$1"
> > +git push
>
> Looks like a private shell script of yours
> has been included by mistake.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nunit/nunit/pull/586/files#r28156075>.
>